src/browser/tab/mod.rs

```diff
@@ -380,4 +377,34 @@ impl<'a> Tab {
         })?;
         Ok(self)
     }
+
+    /// Tells Chrome to start tracking which lines of JS have been executed.
+    ///
+    /// Equivalent to hitting the record button in the "coverage" tab in Chrome DevTools.
+    /// See the file `tests/coverage.rs` for an example.
+    pub fn start_js_coverage(&self) -> Result<&Self, Error> {
+        self.call_method(profiler::methods::Enable {})?;
+
+        self.call_method(profiler::methods::StartPreciseCoverage {
+            call_count: None,
+            detailed: Some(true),
+        })?;
+        Ok(self)
+    }
+
+    /// Tells Chrome to stop tracking which lines of JS have been executed.
+    pub fn stop_js_coverage(&self) -> Result<&Self, Error> {
+        self.call_method(profiler::methods::StopPreciseCoverage {})?;
+        Ok(self)
+    }
+
+    /// Returns info about which lines of JS have been executed since you last called this method
```

Comment:
The original docs on this seem more detailed: `Collect coverage data for the current isolate, and resets execution counters. Precise code coverage needs to have started.`